helpers/lib/cdnify.js

```diff
@@ -16,38 +16,45 @@ module.exports = globals => {
         const editSessionId = siteSettings.theme_session_id;
         const cdnSettings = themeSettings.cdn;
 
-        const protocolMatch = /(.*!?:)/;
-
         if (path instanceof globals.handlebars.SafeString) {
             path = path.string;
         }
 
         if (!path) {
             return '';
         }
-
-        if (/^(?:https?:)?\/\//.test(path)) {
+        
+        if (path.startsWith('//') || path.startsWith('http')) {
```

Comment:
Let's reverse the order here since it'll be more common for the url to start with `http` than `//` at this point

Reply:
fixed.